plugins/CoreHome/angularjs/common/services/piwik.js

```diff
@@ -68,10 +68,10 @@
 
             // Cache server-rendered page title
             originalTitle = originalTitle || document.title;
-            var titleParts = originalTitle.split('-');
-            var dateString = ' ' + piwikPeriods.parse(period, date).getPrettyString() + ' ';
+            var titleParts = originalTitle.split('- ');
```

Comment:
This won't work for page titles like `Brand - Shop`. Not much into that topic atm, but can't we use the globally available `piwik.siteName` to ensure the site name isn't splitted up?

Reply:
I'm completely new to matomo and just tried to fix the wrong title for my own domain rothauer-it.com (which was separated into) "rothauer- Von 2018-11-11 -it.com - Webanalytik-Berichte - Matomo"
My commit fixes that issue.
I have no idea what values the originalTitle can hold besides domain names, so please go ahead and change my PR.